projects/packages/videopress/src/class-block-editor-content.php

```diff
@@ -95,6 +100,7 @@ public static function videopress_embed_shortcode( $atts ) {
 			'controls'        => $atts['controls'],
 			'playsinline'     => $atts['playsinline'],
 			'useAverageColor' => $atts['useaveragecolor'], // The casing is intentional, shortcode params are lowercase, but player expects useAverageColor
+			'preloadContent'  => $atts['preloadcontent'], // The casing is intentional, shortcode params are lowercase, but player expects useAverageColor
```

Comment:
The copied comment here is out of context.

Reply:
Nice catch 😅

Reply:
I hate to nitpick, but the comment still seems to be incorrect.

Reply:
Sorry about that, looks like I missed one.